src/size-list.js

```diff
@@ -41,7 +41,7 @@ let SizeListOptionDef;
  * @param {string} s
  * @return {!SizeList}
  */
-export function parseSizeList(s) {
+export function parseSizeList(s, allowPercentAsLength) {
```

Comment:
Two small things here:

1. Let's rename the argument to `opt_allowPercentAsLength` (`opt_`) suffix.
2. Add another param in the jsdoc: `@param {boolean=} opt_allowPercentAsLength`